README.md

```diff
@@ -95,10 +95,8 @@ Create a simple HTTP API configuration for the client.
 ```golang
 statful.New(
     statful.Configuration{
-
-        AutoFlush: false,
+        DisableAutoFlush: false,
```

Comment:
There's no need to set this - it will be false by default - let's try to keep the examples as simple as possible.